### PR TITLE
Announcement Banner

### PR DESCRIPTION
**Pull request by neil-ptr**
**description**
Adds announcement banner

closes: #130

Screen Shot 2022-06-08 at 3 22 50 PM

### CODE REVIEW

File: libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+export default {
+  component: AnnouncementBanner,
+  title: 'Components/AnnouncementBanner',
+} as Meta;
+
+const Template: Story = (args) => {
+  return (
+    <AnnouncementBanner {...args}>
+      <div className="text-center text-xs font-semibold md:text-sm">
```
Comment (alse) [inline libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx:12]:
```suggestion
      <div className="text-center text-xs font-semibold md:text-sm dark:text-black">
```

Comment (alse) [inline libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx:12]: This needs a `dark:text-black` class. Maybe the styles should be in the AnnouncementBanner?

File: libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        Permissionless deployment is coming to Flow! Read more{' '}
+        <span style={{ textDecoration: 'underline' }}>
+          <a href="https://permissionless.onflow.org/">here</a>
```
Comment (alse) [inline libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx:15]: Should this have a target="_blank" to open a new window? Assuming this component will be added to the site later?

**Review by alse — CHANGES_REQUESTED** [review]

File: libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx (hunk 2 above)
Comment (neilZon) [reply libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx:15]: Yeah that would a good idea, nice catch!

File: libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx (hunk 1 above)
Comment (neilZon) [reply libs/design-system/src/lib/Components/AnnouncementBanner/AnnouncementBanner.stories.tsx:12]: Also a good catch!

**Review by 10thfloor — APPROVED** [review]

**Review by alse — APPROVED** [review]
👍

**Pull request merged**